src/navigation-bar.tsx

```diff
@@ -171,7 +217,7 @@ export const MainNavigationBar: React.FC<MainNavigationBarProps> = ({
                 {showBurger && (
                     <Portal>
                         <FocusTrap disabled={disableFocusTrap} group="burger-menu-lock">
-                            <CSSTransition
+                            <Transition
```

Comment:
CSSTransition was not applying the enter animation at all. I was able to track down the issue to the moment in which we started using React18. Using Transition instead of CSSTransition makes it work fine.